katas/main/src/pokerhands/joaquin/Deck.cs

```diff
@@ -0,0 +1,45 @@
+namespace PokerHand.Joaquin;
+
+using System.Collections.Generic;
+
+public class Deck
+{
+    public string[] cards = new string[52];
+
+    public char[] suits = { 'C', 'D', 'H', 'S' };
+
+    public char[] cardValues = { '2', '3', '4', '5', '6', '7', '8', '9', 'T', 'J', 'Q', 'K', 'A' };
```

Comment:
Fields `cards`, `suits` and `cardValues` in this class seem to be constants, so they should be defined as `const` or `readonly`, or if not accessed outside of this class, declare them as private const or private readonly

Reply:
Done. I declared them as public readonly because I'm using them outside the class.